Libraries/Text/RCTTextView.m

```diff
@@ -633,6 +647,11 @@ - (BOOL)resignFirstResponder
 - (void)layoutSubviews
 {
   [super layoutSubviews];
+
+  // Start sending content size updates only after the view has been layed out
+  // otherwise we send multiple events with bad dimentions on initial render.
+  _sendContentSizeUpdates = YES;
```

Comment:
@javache / @sahrens - can either of you confirm if this is the right thing to do here?